docs/develop/contracts/js/quickstart.md

```diff
@@ -1,66 +1,86 @@
+
+Much like Aurora, we have created a JavaScript Enclave or `JSVM` (JavaScript Virtual Machine) that is also contained within a smart contract deployed on NEAR.
+
+This provides an isolated environment where developers can use a language they are familiar with as well as utilize JavaScript dependencies they are used to. `near-sdk-js` allows you to learn the basics of smart contract development on NEAR without the needing to learn a new programming language. 💪
 
-The JavaScript Enclave, or JavaScript Virtual Machine (jsvm) provides an isolated environment where users can learn the basics of how to write smart contracts on NEAR. This isolated environment is run on a virtual machine, similar to how [Aurora](https://doc.aurora.dev/getting-started/aurora-engine) operates. Standard smart contracts that are built using Rust or AssemblyScript compile to [WebAssembly](https://webassembly.org/) or simply WASM. With the jsvm, contracts are encoded to base64 and are deployed to a virtual machine.
+:::tip Tip
+There are several pros and cons when comparing native `Wasm` smart contracts with `JSVM` smart contracts.
+:::
 
-There are several pros and cons when comparing the enclave approach to the regular WASM approach. The key differences are outlined below.
+| Areas                                      | Native (Wasm) | JSVM        |
+| ------------------------------------------ | ------------- | ----------- |
+| Synchronous Cross-Contract Calls           | ❌            | ✅          |
+| Call methods on any smart contract on NEAR | ✅            | ❌          |
+| Standards Support                          | ✅            | Not in v1.0 |
+| Function Call Access Key Support           | ✅            | Not in v1.0 |
 
-| Areas                     | WebAssembly                                    | Enclave                                                                                                                                                     |
-|---------------------------|--------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| Can interact with any smart contract on NEAR                     |✅|❌|
-| Synchronous Cross-Contract Calls                                 |❌|✅|
-| Standards Support                                                |✅|Not in v1.0|
-| Function Call Access Key Support                                 |✅|Not in v1.0|
+In addition to the ability to develop in a language you are already familiar with, the `JSVM` operates in a synchronous environment which provides an easy way to create smart contracts that interact with other contracts know as performing "cross-contract calls.
```

Comment:
I would mention that these are only supported between other contracts in the enclave.